test/remote/gateways/remote_hps_test.rb

```diff
@@ -3,10 +3,13 @@
 class RemoteHpsTest < Test::Unit::TestCase
   def setup
     @gateway = HpsGateway.new(fixtures(:hps))
+    @check_gateway = HpsGateway.new(fixtures(:hps_echeck))
 
     @amount = 100
+    @check_amount = 2000
     @declined_amount = 1034
     @credit_card =   credit_card('4000100011112224')
```

Comment:
Interesting spacing on this line :) (I know your PR doesn't touch this, just jumped out at me).

Reply:
Rubocop will come for it one of these days, I'm sure :D